outport/outport.go

```diff
@@ -0,0 +1,133 @@
+package outport
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/core/statistics"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/state"
+	"github.com/ElrondNetwork/elrond-go/outport/drivers"
+	"github.com/ElrondNetwork/elrond-go/outport/types"
+)
+
+type proxy struct {
```

Comment:
Rename to `outportProxy` or just `outport`?

Reply:
`outport` as to comply with the functionality, filename and constructor function

Reply:
done.